### DISCUSSION TITLE
TShock version for Crossplay?

### DISCUSSION THREAD

**Discussion by Eduxious** [Pryaxis/TShock#2855] [Q&A] [answered]
Hi! I'm trying to set up a Cross play server for PC and Mobile using TShock, and I already have the Crossplay.dll but there's no way of us being able to join each other because the server software that TShock uses is higher than the versions we play on.
Since the Latest version in Mobile is 1.4.4.5, I also installed the desktop counterpart for that, but there is no TShock server running on the said version? 
Will there ever be a TShock server software for this version? Or can anyone teach me how to port the server to the said version. Thank you!!

**Comment by hakusaro** [comment]
> Hi! I'm trying to set up a Cross play server for PC and Mobile using TShock, and I already have the Crossplay.dll but there's no way of us being able to join each other because the server software that TShock uses is higher than the versions we play on.

This is really a question for the author of Crossplay.

> Will there ever be a TShock server software for this version?

No.

**Comment by sgkoishi** [comment, marked as answer]
Currently both Terraria PC, Mobile and TShock are all at 1.4.4.9.
So, if all of you are using the latest version, it should work without Crossplay.

**Reply by sgkoishi** [reply]
Just double-checked and yes, [PC, Android, iOS, Xbox, and PlayStation are 1.4.4.9. The Switch and Amazon store are still behind but should be available soon.](https://forums.terraria.org/index.php?threads/terraria-labor-of-love-is-out-now.114357/post-2832196)

**Reply by Eduxious** [reply]
Hi! Thanks I just noticed! It just dropped literally yesterday and I didn't notice. Thank you so much!
